src/sunburst.js

```diff
@@ -23,6 +20,8 @@ export default Kapsule({
     sort: { onChange(_, state) { state.needsReparse = true }},
     label: { default: d => d.name },
     labelOrientation: { default: 'auto' }, // angular, radial, auto
+    textSize: {default: 12},
```

Comment:
May be more fitting to call these two props `labelFontSize` and `labelStrokeWidth`.

Reply:
I will have a look later, and follow your suggestions. Thank you for having a look!